app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/network/api/IdentityApi.kt

```diff
@@ -66,4 +69,14 @@ interface IdentityApi {
 
     @POST("/accounts/register")
     suspend fun register(@Body body: RegisterRequestJson): Result<RegisterResponseJson.Success>
+
+    @POST("/accounts/register/finish")
+    suspend fun registerFinish(
+        @Body body: RegisterFinishRequestJson,
+    ): Result<RegisterResponseJson.Success>
+
+    @POST("/accounts/register/send-verification-email")
+    suspend fun sendVerificationEmail(
+        @Body body: SendVerificationEmailRequestJson,
+    ): Result<ResponseBody?>
```

Comment:
We discussed this offline, some changes have gone into the app to better support nullable values.

You can update this to return a `String?` directly now